controllers/om/automation_config_test.go

```diff
@@ -627,6 +628,54 @@ func TestAutomationConfigEquality(t *testing.T) {
 	}
 	ldapConfig2 := ldapConfig
 
+	oidcProviderConfigs := []oidc.ProviderConfig{
+		{
+			AuthNamePrefix:        "provider-1",
```

Comment:
Since the implementation implies that we support repeating the `AuthNamePrefix`, we should have a test displaying the expected behavior.

Reply:
Is it ok to resolve the comment? @mircea-cosbuc @lsierant

Reply:
Ok from my side, I didn't resolve it because of @lsierant 's comment.